README.md

```diff
@@ -56,6 +56,8 @@ The library also provides integrations with all the payment methods supported by
 
 Using this library in conjunction with your favorite UI framework, it is possible to build a bespoke checkout UI for a store, that can be augmented with additional features. We provide a basic [reference implementation](https://github.com/bigcommerce/checkout-sdk-js-example) of a custom checkout written in React to get you started.
 
+## Limitations
+* It is not possible to execute custom jscript when paying on hosted payment pages such as Adyen, therefore Google Enhenced eCommerce Analytics can not be fully used as payment confirmation is handled by payments.bigcommerce.com and not by the checkout.
```

Comment:
```suggestion
* It is not possible to execute custom JavaScript when paying on hosted payment pages such as Adyen, therefore Google Enhanced eCommerce Analytics can not be fully used as payment confirmation is handled by payments.bigcommerce.com and not by the checkout.
```

Could you please provide some context on what you mean by _Google Enhanced eCommerce Analytics **can not be fully used**_? Thanks 🙇

Reply:
@icatalina Sorry, I just saw this comment. You are right, the more precise description would be

It is not possible to execute custom JavaScript when paying on hosted payment pages such as Adyen on the checkout after the payment has been completed, therefore conversion tracking solutions or any other task that needs to be completed after the payment need to implemented on the order confirmation page.